code/modules/mob/mob_helpers.dm

```diff
@@ -113,6 +113,24 @@
 		return U.sensor_mode
 	return SUIT_SENSOR_OFF
 
+/proc/offer_control(mob/M)

```

Comment:
Shouldn't this be a `mob/living` proc?

Reply:
pretty sure mob/M is the mob being offered control